src/routes/console/organization-[organization]/settings/deleteOrganizationModal.svelte

```diff
@@ -51,7 +53,7 @@
         }
     }
 
-    $: upcomingInvoice = invoices.invoices.find((i) => i.status === 'upcoming' && i.amount > 0);
+    $: upcomingInvoice = invoices?.invoices.find((i) => i.status === 'upcoming' && i.amount > 0);
```

Comment:
```suggestion
    $: upcomingInvoice = invoices?.invoices?.find((i) => i.status === 'upcoming' && i.amount > 0);
```

I think a second `?` will be necessary, otherwise, it becomes `null.find()`.  I think, I might be totally wrong.

Reply:
I don't think so, `Cloud` always returns an empty list if there are no invoices and returns `undefined` if on self-hosted.

---
Here's an actual return from the api -
URL - `https://cloud.appwrite.io/v1/organizations/{:orgId}/invoices?organizationId={:orgId}`
Response -
```json
{
  "total": 0,
  "invoices": []
}
```

Let me know if I should anyway add a second `?`.

Reply:
You are right, seems to be fine as it is.
![CleanShot 2024-04-16 at 10 57 22@2x](https://github.com/appwrite/console/assets/19310830/f382735c-28af-4b79-a464-9273b3a4a560)